Changelog.md

```diff
@@ -1,6 +1,11 @@
 ### 0.6.10 (unreleased)
 
+Language Features:
+
+Compiler Features:
 
+Bugfixes:
+* Type system: Fix internal compiler error on calling externally a function that returns calldata pointers
```

Comment:
```suggestion
* Type system: Fix internal compiler error on calling externally a function that returns calldata pointers.
```

Reply:
maybe "that returns variables with calldata location"?

Reply:
Yes, it makes more sense to write "that returns variables with calldata location"